src/posh-winfetch.ps1

```diff
@@ -22,6 +22,22 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+<#PSScriptInfo
+.VERSION 1.2.1
+.GUID 1c26142a-da43-4125-9d70-97555cbb1752
+.DESCRIPTION Winfetch is a command-line system information utility for Windows written in PowerShell.
+.AUTHOR lptstr
+.PROJECTURI https://github.com/ppadial/winfetch
```

Comment:
That's the wrong URL ;(

Reply:
True, i forgot to change it i created it using an script that took the info from the repository